readhoney/views/index.pug

```diff
@@ -5,9 +5,10 @@ block content
   h1= title
   h1 Welcome to #{title}
   h3 You ARE HERE
+  
+  a(href='/logout') Logout 
```

Comment:
Right now, the code is having the effect we want it to. When you come over to us, maybe we can go over this more in depth. We are keeping this branch and the pull request open in the meantime.